internal/rbd/rbd_util.go

```diff
@@ -893,7 +882,9 @@ func (rv *rbdVolume) updateVolWithImageInfo(cr *util.Credentials) error {
 	// rbd --format=json info [image-spec | snap-spec]
 	var imgInfo imageInfo
 
-	stdout, stderr, err := util.ExecCommand("rbd",
+	stdout, stderr, err := util.ExecCommand(
+		context.TODO(),
```

Comment:
pass the ctx to `updateVolWithImageInfo` which helps to log request ID

Reply:
this is to be re-written with the go-ceph API, and in that case no command will get executed anymore. The context.Context() will then get removed. The only caller (`getImageInfo()`) would need to get the context passed as well. I do not think this is worth the effort.

Reply:
Sounds good